docs/src/main/asciidoc/kafka-reactive-getting-started.adoc

```diff
@@ -0,0 +1,365 @@
+
+The method returns a _Reactive Stream_.
+The generated items are sent to the stream named `generated-price`.
+This stream is mapped to Kafka using the `application.properties` file that we will create soon.
+
+== The price converter
+
+The price converter reads the prices from Kafka, and transforms them.
+Create the `src/main/java/org/acme/kafka/PriceConverter.java` file with the following content:
+
+[source, java]
+----
+package org.acme.kafka;
+
+import org.eclipse.microprofile.reactive.messaging.Incoming;
+import org.eclipse.microprofile.reactive.messaging.Outgoing;
+
+import javax.enterprise.context.ApplicationScoped;
+
+/**
+ * A bean consuming data from the "prices" Kafka topic and applying some conversion.
+ * The result is pushed to the "my-data-stream" stream which is an in-memory stream.
+ */
+@ApplicationScoped
+public class PriceConverter {
+
+    private static final double CONVERSION_RATE = 0.88;
+
+    @Incoming("prices")                                     // <1>
+    @Outgoing("my-data-stream")                             // <2>
+    public Price process(Price priceInUsd) {
+        return new Price(priceInUsd.value * CONVERSION_RATE, "€");
+    }
+
+}
+----
+<1> Indicates that the method consumes the items from the `prices` topic
+<2> Indicates that the objects returned by the method are sent to the `my-data-stream` stream
+
+The `process` method is called for every Kafka _record_ from the `prices` topic (configured in the application configuration).
+Every result is sent to the `my-data-stream` in-memory stream.
+
+== The price resource
+
+Let's bind our stream to a JAX-RS resource and send events to clients.
+Create the `src/main/java/org/acme/kafka/PriceResource.java` file with the following content:
+
+[source,java]
+----
+package org.acme.kafka;
+
+import javax.inject.Inject;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+import org.eclipse.microprofile.reactive.messaging.Channel;
+import org.reactivestreams.Publisher;
+
+/**
+ * A simple resource retrieving the prices topic and sending the items to a server sent event.
+ */
+@Path("/prices")
+public class PriceResource {
+
+    @Inject
+    @Channel("my-data-stream") Publisher<Price> prices; // <1>
+
+    @GET
+    @Path("/stream")
+    @Produces(MediaType.SERVER_SENT_EVENTS) // <2>
+    public Publisher<Price> stream() { // <3>
+        return prices;
+    }
+}
+----
+<1> Injects the `my-data-stream` channel using the `@Channel` qualifier
+<2> Indicates that the content is sent using `Server Sent Events`
+<3> Returns the stream (_Reactive Stream_)
+
+== JSON serialization via Jackson
+
+Finally, we will configure JSON serialization for messages using Jackson.
+Quarkus provides default implementations for Kafka serializer/deserializer pairs using Jackson `ObjectMapper`.
+`ObjectMapperSerializer` can be used to serialize all pojos via Jackson.
```

Comment:
I would replace "pojos", but I don't have a better name. 
(pojo is not used anymore)

Reply:
I went with "objects"